common/api/presentation-frontend.api.md

```diff
@@ -136,7 +136,7 @@ export enum FavoritePropertiesScope {
     // (undocumented)
     IModel = 2,
     // (undocumented)
-    Project = 1
+    ITwin = 1

```

Comment:
This will need to be in the NextVersion.md as a breaking change

Reply:
@grigasp

Reply:
I agree - `FavoritePropertiesScope.Project` should be added to removed APIs table with `FavoritePropertiesScope.ITwin` specified as replacement.

Reply:
I agree - `FavoritePropertiesScope.Project` should be added to removed APIs table with `FavoritePropertiesScope.ITwin` specified as replacement.

Reply:
Updated the NextVersion.md with the change.